### PR TITLE
(2596) Remove staff namespace

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Remove the "staff" namespace

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Remove `staff` namespace](https://trello.com/c/onxxOoHa/2596-remove-staff-namespace)

**Comment by Gweaton** [comment]
Will the removal of the files in `views/staff` come in a separate PR?

File: spec/views/staff/exports/organisations/show_spec.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-RSpec.describe "staff/exports/organisations/show" do
```
Comment (yndajas) [inline spec/views/staff/exports/organisations/show_spec.rb:1]: `spec/views/staff/exports` is still namespaced under `staff`

**Comment by Gweaton** [comment]
> > Will the removal of the files in `views/staff` come in a separate PR?
> 
> I think it is removed - when I look at this locally the directory isn't there, it's just listed in the diff because it's all been deleted

Ah sorry, it was the spec files I was looking at locally:

image

**Review by yndajas — APPROVED** [review]
🎉🎉🎉

I'm assuming the removed Brakeman exception for the `UsersController` is no longer needed since the tests are passing now?

**Pull request merged**